### DISCUSSION TITLE
Admin Policies in Google Workspace?

### DISCUSSION THREAD

**Discussion by zpoling** [uBlockOrigin/uBOL-home#111] [Q&A] [answered]
I'm looking for some help pushing out uBOL with the JSON admin policies via Google Workspace. I'm really just trying to push the _disableFirstRunPage_ setting for now.

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uBOL-home/wiki/Managed-settings

**Reply by zpoling** [reply]
That doesn't appear to work, or at least it doesn't when applying the JSON setting directly in Google Workspace. I haven't tested with GPO settings ([discussed here](https://github.com/uBlockOrigin/uBOL-home/discussions/35)) as I'm looking to push the extension to ChromeOS. The JSON value does not appear in chrome://policy under uBOL.

In the same thread, I saw it mentioned that it may take a few restarts of the browser/ reloading policies before the JSON values populate to end users. This makes _disableFirstRunPage_ a bit useless as the page will have already appeared before the JSON values are applied.

**Reply by zpoling** [reply]
I got closer with this. It's appearing in chrome://policy now, but it isn't being recognized and is subsequently ignored.

`{
   "disableFirstRunPage": {
      "Value": [
         "true" 
      ]
   }
}`

![image](https://github.com/uBlockOrigin/uBOL-home/assets/157506869/75bab597-24a0-4048-b677-bde69fa3e522)

**Reply by gorhill** [reply]
The value is of type boolean, not string. Use `true` instead of `"true"`. The example at the link above shows that.

**Reply by zpoling** [reply, marked as answer]
Not entirely sure what happened. The following is working now when it wasn't before; this is what I had set when I took the screencap of chrome://policy. I think I may have had a space somewhere I shouldn't have.

```
{ "disableFirstRunPage": {
      "Value": true
   }
}
```

And just to have both applied in Workspace:
```
{ "disableFirstRunPage": {
      "Value": true
   },
 "noFiltering": {
      "Value": [ 
          "domain.com" 
      ]
   }
} 
```

**Reply by gorhill** [reply]
>  I think I may have had a space somewhere I shouldn't have.

You were using quotes:

> `{ "disableFirstRunPage": { "Value": [ "true"  ] } }`

**Reply by zpoling** [reply]
At that time, yes, but I wasn't at the time of taking the screen capture. With what you quoted, it appeared as "true" in chrome://policy. Now my final settings match what is shown in the screen capture, but it is working unlike before.
